openqa-prepare-mm-setup

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env perl
+# Copyright 2020 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+use strict;
+
+system("zypper in -y os-autoinst-openvswitch");
+system("systemctl enable --now os-autoinst-openvswitch");
+
+my $bridge = 1;
+my $openvswitch = "/etc/sysconfig/os-autoinst-openvswitch";
+open(file, '>', $openvswitch) or die "Failed to open $openvswitch for writing!";
+print file "OS_AUTOINST_USE_BRIDGE=br$bridge";
+close(file);
```

Comment:
Also these calls need error handling (unless autodie is used).

Reply:
+1 for autodie